lib/node_modules/@stdlib/constants/float32/ln-half/README.md

```diff
@@ -0,0 +1,135 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# FLOAT32_LN_HALF
+
+> [Natural logarithm][@stdlib/math/base/special/ln] of single-precision floating-point number `1/2`.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var FLOAT32_LN_HALF = require( '@stdlib/constants/float32/ln-half' );
+```
+
+#### FLOAT32_LN_HALF
+
+[Natural logarithm][@stdlib/math/base/special/ln] of `1/2`.
```

Comment:
@vivekmaurya001 Also, you'll need to update this path to the single-precision equivalent, i.e., `lnf` here and everywhere else in this file.

Reply:
done it !